src/client/pages/Welcome.stories.tsx

```diff
@@ -1,50 +1,43 @@
 import React from 'react';
-import { Meta } from '@storybook/react';
+import { ComponentMeta, Story } from '@storybook/react';
 
 import { Welcome } from './Welcome';
 
 export default {
   title: 'Pages/Welcome',
   component: Welcome,
   parameters: { layout: 'fullscreen' },
-} as Meta;
+} as ComponentMeta<typeof Welcome>;
 
-export const Default = () => <Welcome submitUrl="" fieldErrors={[]} />;
-Default.story = {
-  name: 'with defaults',
-};
+type PartialProps = Partial<React.ComponentProps<typeof Welcome>>;
 
-export const Email = () => (
-  <Welcome submitUrl="" email="example@theguardian.com" fieldErrors={[]} />
+const Template: Story<PartialProps> = (props) => (
+  <Welcome submitUrl="" fieldErrors={[]} {...props} />
 );
-Email.story = {
-  name: 'with email',
-};
 
-export const FieldErrorPW = () => (
-  <Welcome
-    submitUrl=""
-    email="example@theguardian.com"
-    fieldErrors={[
-      {
-        field: 'password',
-        message: 'Password must be between 8 and 72 characters.',
-      },
-    ]}
-  />
-);
-FieldErrorPW.story = {
-  name: 'with error on password',
+export const Default = Template.bind({});
+Default.storyName = 'with defaults';
+
+export const Email = Template.bind({});
+Email.args = { email: 'example@theguardian.com' };
+Email.storyName = 'with email';
+
+export const FieldErrorPW = Template.bind({});
+FieldErrorPW.args = {
+  email: 'example@theguardian.com',
+  fieldErrors: [
+    {
+      field: 'password',
+      message: 'Password must be between 8 and 72 characters.',
+    },
+  ],
 };
+FieldErrorPW.storyName = 'with error on password';
 
-export const PasswordAlreadySet = () => (
-  <Welcome
-    submitUrl=""
-    email="example@theguardian.com"
-    fieldErrors={[]}
-    passwordSet={true}
-  />
-);
-PasswordAlreadySet.story = {
-  name: 'with password already set',
+export const PasswordAlreadySet = Template.bind({});
+PasswordAlreadySet.args = {
+  email: 'example@theguardian.com',
+  submitUrl: '',
+  passwordSet: true,
 };
+PasswordAlreadySet.storyName = 'with password already set';
```

Comment:
Just amending this story so you can easily change the parameters via the action tab in storybook.